.github/workflows/ci.yml

```diff
@@ -55,9 +55,11 @@ jobs:
         run: "composer code-style:check"
 
       - name: "Run PHPStan"
+        if: ${{ matrix.dependencies == 'locked' }}
         run: "composer phpstan"
 
       - name: "Run Rector"
+        if: ${{ matrix.dependencies == 'locked' }}
```

Comment:
We could probably also be running rector only on single version of PHP, as we bind it in the config anyway..? 🤔

Reply:
Yeah I think that's reasonable, let's run @ 8.1?